src/main/java/com/org/trustservice/model/Department.java

```diff
@@ -0,0 +1,49 @@
+package com.org.trustservice.model;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+import java.util.UUID;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.annotations.GenericGenerator;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "department")
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class Department {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name = "uuid", strategy = "org.hibernate.id.UUIDGenerator")
+    private UUID deptId;
```

Comment:
Random ID generator for database to be avoided. 
UUID is good for generating random values like codes, otps, file names,  etc. But not ids.